build/yaml/deployBotResources/js/evaluateDependenciesVariables.yml

```
@@ -22,23 +22,29 @@ steps:
         $sourceJSMyGet = "https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/"
         $sourceJSv3MyGet = "https://botbuilder.myget.org/F/botbuilder-v3-js-daily/npm/"
         $sourceJSNpm = "https://registry.npmjs.com/"
-        switch -regex ("${{ parameters.registry }}") {
+        $versionNumber = "${{ parameters.version }}"
+        $registry = "${{ parameters.registry }}"
+        $botType =  "${{ parameters.botType }}"
+        switch -regex ($registry) {
           "^($null|MyGet)$" {
-            switch ("${{ parameters.botType }}") {
+            switch ($botType) {
               "SkillV3" { $source = $sourceJSv3MyGet }
               default { $source = $sourceJSMyGet }
             }
           }
-          "Npm" { $source = $sourceJSNpm }
-          default { $source = "${{ parameters.registry }}" }
+          "Npm" { $source = $sourceJSNpm 
+            if ([string]::IsNullOrEmpty($versionNumber)){
+              $versionNumber = "STABLE"
+            }
+          }
```

Please, format these lines
```suggestion
          "Npm" { 
            $source = $sourceJSNpm 
            if ([string]::IsNullOrEmpty($versionNumber)){
              $versionNumber = "STABLE"
            }
          }
```

---

Fixed in a new commit